static/js/lib/ecommerce.js

```diff
@@ -33,3 +34,6 @@ export const formatPrice = (price: ?string | number | Decimal): string => {
     return `$${formattedPrice}`
   }
 }
+
+export const isPromo = (couponType: string): boolean =>
```

Comment:
Take it or leave it: could be a ramda `equals(COUPON_TYPE_PROMO)`

Reply:
✔️